tough/src/http.rs

```diff
@@ -172,7 +172,7 @@ impl RetryState {
                 Ordering::Greater => {
                     self.wait = settings.max_backoff;
                 }
-                _ => {}
+                Ordering::Equal => {}
```

Comment:
Does the new lint not allow the `_`?

Reply:
My guess would be that it does not allow it for enums that have only two cases.

Reply:
The error seems to indicate it doesn't allow `_` *if* Clippy can determine that it could be trivially replaced with a single variant. https://rust-lang.github.io/rust-clippy/master/index.html#match_wildcard_for_single_variants

If you're already matching multiple variants it doesn't seem to mind that.